ct/headscale.sh

```diff
@@ -41,7 +41,9 @@ function update_script() {
     msg_ok "Updated $APP to ${RELEASE}"
 
     msg_info "Starting ${APP}"
-    systemctl enable --now headscale
+    # Temporary fix until headscale project resolves service getting disabled on updates.
+    systemctl -q enable headscale
+    systemctl start headscale
```

Comment:
```suggestion
    systemctl enable -q --now headscale
```
2 separate commands are not needed

Reply:
my suggestion was before Mick told you just to add `-q --now` which is how we usually do it. in that case 2 separate commands are not needed. anyway, merged